test/distributed/_shard/sharded_tensor/ops/test_matrix_ops.py

```diff
@@ -158,9 +132,6 @@ def _test_masked_fill_with_sizes(self, mask_size, broadcast_style=False):
                 mask = mask.unsqueeze(1)
             tensor_m = tensor.masked_fill(mask, 25.0)
             st_expected = _shard_tensor(tensor_m, spec)
-            st_expected._metadata.shards_metadata.sort(
-                key=lambda x: x.shard_offsets[0],
```

Comment:
why we don't need sort this time?

Reply:
Because your PR(https://github.com/pytorch/pytorch/pull/75991) has fixed it so we now don't need it anymore.